pixelmod/src/main/java/org/e8yes/pixelmod/renderer/IDecalRenderer.kt

```diff
@@ -8,5 +8,5 @@ import org.e8yes.pixelmod.repository.DecalHandle
  * possible.
  */
 interface IDecalRenderer {
-    fun render(src: Bitmap, decal: DecalHandle): Bitmap
+    fun render(src: Bitmap, detect: DecalHandle, mod: DecalHandle): Bitmap
```

Comment:
can we use something like `srcLogo` and `destLogo`?
or update the kdoc with @param detect, @param mod to explain what they are?

Reply:
ok